lang/java/reef-runtime-azbatch/src/test/java/org/apache/reef/runime/azbatch/CommandBuilderTests.java

```diff
@@ -0,0 +1,69 @@
+package org.apache.reef.runime.azbatch;
```

Comment:
This file is using 4 indents format. While most other files in reef are using 2 indents format.

Reply:
Fixed it here and in my Intellij settings.